ui/main/src/app/services/customScreen/CustomScreenService.ts

```
@@ -0,0 +1,24 @@
+    private static customScreenDefinitions = new Map<string, CustomScreenDefinition>();
+
+    public static addCustomScreenDefinition(customScreenDefinition: CustomScreenDefinition) {
+        CustomScreenService.customScreenDefinitions.set(customScreenDefinition.id, customScreenDefinition);
+    }
+    public static getCustomScreenDefinition(customScreenId: string): CustomScreenDefinition {
+        return CustomScreenService.customScreenDefinitions.get(customScreenId);
+    }
+    public static clearCustomScreenDefinitions() {
+        CustomScreenService.customScreenDefinitions.clear();
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)